expression/builtin_time_test.go

```diff
@@ -471,6 +471,36 @@ func (s *testEvaluatorSuite) TestStrToDate(c *C) {
 	}
 }
 
+func (s *testEvaluatorSuite) TestFromDays(c *C) {
```

Comment:
Hi, please add some test cases for string and negative numbers.
like "149a" "366a" "abc"